roles/openshift_cli/defaults/main.yml

```diff
@@ -1,2 +1,2 @@
 ---
-openshift_version: "{{ openshift_image_tag | default(openshift.common.image_tag) | default('') }}"
+openshift_version: "{{ openshift_image_tag | default(openshift.docker.openshift_image_tag) | default('') }}"
```

Comment:
Good point. As long as we haven't shipped a version that uses that fact, it should be fine to move it without migration.